var/spack/repos/builtin/packages/acts-core/package.py

```diff
@@ -52,22 +55,20 @@ class ActsCore(CMakePackage):
     variant('tgeo', default=False, description='Build the TGeo plugin')
 
     depends_on('cmake @3.7:', type='build')
-    depends_on('boost @1.62: +program_options +test')
+    depends_on('boost @1.62:1.69.99 +program_options +test')
```

Comment:
Please add comment on why this is incompatible with boost@1.70.  Please provide URL is such a discussion exists online.

Reply:
Done, thanks for the review!